OasisDEX/oasis-borrow
# pick trigger with highest stop loss level

# [Pick trigger with highest stop loss level](https://shortcutLink)

<please insert a shortcut link above>
  
## Changes 👷‍♀️
  <Please add short list of changes>
- if there are more than 1 active trigger added to vault, logic from this PR ensure that we will pick trigger with highest stop loss level
  
## How to test 🧪
  <Please explain how to test your changes>
- add two triggers to existing vault using hardhat task, check if trigger with highest value is presented on the UI
